### PR TITLE
Upgrades storybook to v5

### PR DESCRIPTION
**Pull request by tinkertrain**
Storybook v5 has a bunch of nice things added, among them a much nicer UI and a few extra things. More info [here](https://medium.com/storybookjs/storybook-5-0-db1d0f9c83b8)

Followed the instructions found here:
- https://medium.com/storybookjs/storybook-5-migration-guide-d804b38c739d
- https://github.com/storybooks/storybook/blob/next/MIGRATION.md#from-version-41x-to-50x

Screenshot

## Upgrading notes

Upgrade went relatively smooth, I did find a couple of issues with dependencies. Below are some related issues I found and by following some suggestions inside the conversations there, I managed to get v5 running without further issues:

- https://github.com/storybooks/storybook/issues/5706
- https://github.com/storybooks/storybook/issues/5663

Fixes outside the migration guide:

- Adding `emotion-theming@10` to devDependencies
- Adding `@storybook/theming@latest` to devDependencies (in theory it is included as a dependency in the storybook package, but I found that it has to be added)

### CODE REVIEW

**Comment by ItsJonQ** [comment]
@tinkertrain 😍 

So excited for this!!!!!

File: package.json [hunk 1]
```diff
@@ -20,7 +20,8 @@
     "Lauren Brown",
     "Pierre-Luc Babin",
     "Buzz Usborne <hello@buzzusborne.com>",
-    "Alisdair McDiarmid"
+    "Alisdair McDiarmid",
+    "Juan Pablo Lomeli Diaz <juanpablo@thetinkertrain.codes>"
```
Comment (ItsJonQ) [inline package.json]: @tinkertrain 💯 👍

File: package.json [hunk 2]
```diff
@@ -163,6 +164,7 @@
     "css-loader": "0.28.4",
     "cypress": "^3.1.5",
     "dotenv": "4.0.0",
+    "emotion-theming": "^10.0.10",
```
Comment (ItsJonQ) [inline package.json]: Wow. I'm surprised an Emotion 10 package didn't blow up Fancy! (which is on Emotion 9)

Comment (tinkertrain) [reply package.json]: I was worried about that too and for a moment thought it could be a blocker... but seems fine 🤞

**Comment by ItsJonQ** [comment]
@tinkertrain I made a tiny update. I added @TerrenceLJones to `contributors` in the `package.json` as well 😊

**Comment by ItsJonQ** [comment]
Hmm.. wait a sec. Netlify failed now. But why

**Comment by ItsJonQ** [comment]
Lemme try rebuilding package-lock.json?

**Comment by ItsJonQ** [comment]
Hmm. Nope. Pushing my `npm install` generated `package-lock.json` didn't work. Ugh

**Comment by plbabin** [comment]
> Hmm. Nope. Pushing my `npm install` generated `package-lock.json` didn't work. Ugh

might be an issue with netlify. #594 is failing as well

**Comment by ItsJonQ** [comment]
@plbabin Maybe... both of those PRs were done by @tinkertrain though. I'm weeding out the chance that.. somehow.. Juan Pablo's machine did something to `package-lock.json` that Netlify didn't like

We experienced something like that in the application

**Comment by ItsJonQ** [comment]
Wut Netlify. Come on!

**Comment by tinkertrain** [comment]
[Upgrade HSDS React to Storybook 5](https://trello.com/c/BMb9iEFF/187-upgrade-hsds-react-to-storybook-5)

**Comment by ItsJonQ** [comment]
@knicklabs An Artboard feature I really like is the ability to drag/resize the canvas. Handy for responsive testing and for things like Dropdown :).

I think perhaps I can slim down and adjust Artboard for Storybook 5 use. That way, the features compliment vs. clash

**Pull request merged**